File: python/letter_search.py
def letter_search(some_string):
    alphabet = 'abcdefghijklmnopqrstuvwxyz'
    binary_output = ''
    for char in alphabet:
        if char in some_string.lower():
            binary_output += '1'
        else:
            binary_output += '0'
    return binary_output

print(letter_search('Abc'))
            

File: JavaScript/letterSearch.js
const letterSearch = (someString) => {
    const alphabet = "abcdefghijklmnopqrstuvwxyz";
    let binaryOutput = "";

    for (letter of alphabet) {
        if (someString.toLowerCase().includes(letter)) {
            binaryOutput += "1";
        } else {
            binaryOutput += "0";
        }
    }
    return binaryOutput;
};

console.log(letterSearch('ABCDkkDsl'))